lang/python/python-package-install.sh

```diff
@@ -11,6 +11,7 @@ process_filespec() {
 	local dst_dir="$2"
 	local filespec="$3"
 	echo "$filespec" | (
+	set -e
```

Comment:
i'm curious if you have a reproducing case;
this looks like a good idea;

would doing a `export SHELLOPTS` also work ?
that way, if we decide to also enable tracing via `set -x` in the parent script it would get inherited as well